docs/tutorial/new-tutorial.md

```diff
@@ -0,0 +1,384 @@
+(tutorial)=
+# SQLair tutorial
+
+Imagine you are writing a Go program that handles your customers and their
+orders. The business logic is contained in the Go program but the customer and
+order data is stored in a SQL database. In this tutorial, we'll see how easy
+SQLair makes it to map the database between the database and the Go program.
+Let's get started!
+
+
+```{note}
+This tutorial will take around 30 minutes to complete. If you want a quicker
+start see {ref}`prepare-and-run`.
+```
+
+## Setting up the environment
+
+To start with create a new folder called tutorial and initialise a new Go project.
+```bash
+mkdir tutorial 
+cd tutorial
+go mod init sqlair-tutorial
+```
+
+We first need to set up the backbone of our project, the database. The easiest
```

Comment:
This tutorial "we" sounds like a good idea but it's usually either awkward (almost always) or confusing (e.g., when you need to distinguish between "here's the output I got -- yours might be similar"). I'd mostly stick to "you".

Reply:
Cool, will change that.